atividade03/src/main/java/br/edu/uepb/eda/atividade03/PilhaComLista.java

```diff
@@ -0,0 +1,47 @@
+package br.edu.uepb.eda.atividade03;
+
+public class PilhaComLista implements Pilha_IF {
+
+    public int limite = 10;
```

Comment:
como é um valor constante poderia ser reescrito assim: 
```suggestion
    public static int LIMITE = 10;
```